vaadin-grid-flow-parent/vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/editor/EditorRenderer.java

```diff
@@ -178,13 +178,10 @@ private void setupEditorRenderer(Element container,
 
                 // If editing, render the editor, otherwise use the original renderer
                 "if (root.__editing) { Vaadin.FlowComponentHost.setChildNodes('" + appId + "', [model.item._" + columnInternalId + "_editor], root); }" +
-                "else if (!originalRender) { root.textContent = model.item." + columnInternalId + " }" +
+                "else if (!originalRender) { this._defaultRenderer(root, container, model); }" +
                 "else { originalRender(root, container, model); }" +
```

Comment:
**nit:** swap to avoid negative conditions

```suggestion
                "else if (originalRender) { originalRender(root, container, model) }" +
                "else { this._defaultRenderer(root, container, model) }" +
```